source/guides/stacks/dlrs-inference.rst

```diff
@@ -0,0 +1,1035 @@
+.. _dlrs-inference:
+
+AI Inference with the Deep Learning Reference Stack
+###################################################
+
+In this guide we explore a solution for using the Deep Learning Reference Stack with a Seldon Core\* platform that has been deployed on Kubernetes\*.
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The solution covered here uses the following software components:
+
+* `Deep Learning Reference Stack`_ which is a |CL-ATTR| based Docker\* container providing deep learning frameworks and is optimized for Intel Xeon Scalable platforms.
+* `Kubeflow`_ is the machine learning toolkit for Kubernetes that helps with deployment of Seldon Core and Istio components.
+* `Seldon Core`_ is a software platform for deploying machine learning models. We use the DLRS container to serve the OpenVino\* framework for inference with the Seldon Core.
+* The OpenVino Model Server is included in DLRS and provides the OpenVino framework for inference. From OpenVino, the `OpenVino Toolkit`_ provides improved neural network performance on a variety of Intel processors. For this guide, we converted pre-trained Caffe models into the `Intermediate Representation(IR)`_ of ResNet50 with the OpenVino toolkit.
+* `Istio`_ is a traffic manager and performs the load balancing for service requests in the cluster.
+* Pre-processing container converts jpeg content into NumPy array
+* Post-processing container converts an array of classification probabilities to a human-readable class name.
+* Pre and post-processing containers are created using the `Source-to-Image`_ (S2I)toolkit, which builds reproducible container images from source code/
+* `Min.io`_ is used as the distributed object storage for models
+
+.. figure:: /_figures/stacks/kubeflow-seldon-dlrs-example-diagram.png
```

Comment:
Could you move up this figure to precede the Overview and increase its a size a little more? 
It's a neat graphic, but it gets lost at its current location.

Reply:
Good suggestion, latest commit addresses this.